thirdparty/haw_logger/logger/logdefdest.cpp

```diff
@@ -7,15 +7,16 @@
     #define USE_QT_DIR
     #include <QDir>
 #else
-    #if __has_include(<filesystem>)
-        #include <filesystem>
-        namespace fs = std::filesystem;
-    #elif __has_include(<experimental/filesystem>)
-        #include <experimental/filesystem>
-        namespace fs = std::experimental::filesystem;
-    #else
-        #error compiler must either support c++17
-    #endif
+//    #if __has_include(<filesystem>)
+//        #include <filesystem>
+//        namespace fs = std::filesystem;
+//    #elif __has_include(<experimental/filesystem>)
+//        #include <experimental/filesystem>
+//        namespace fs = std::experimental::filesystem;
+//    #else
+//        #error compiler must either support c++17
+//    #endif
+#error At moment supported only with Qt
```

Comment:
With Qt?

Reply:
yes, at the moment using QDir   
(The original idea was not to depend on Qt)

Reply:
needs to define HAW_LOGGER_QT_SUPPORT and have QDir 
(we have it)

Reply:
Ah, I see (now)